README.md

```diff
@@ -19,72 +23,92 @@ yarn create testcafe
 pnpm create testcafe
 ```
 
-## Options
-
-The `create-testcafe` tool accepts the following options:
-
-### run-wizard
+## Specify project name and location
 
-If you enable the `--run-wizard` option, the `create-testcafe` tool will display an interactive wizard that allows you to specify project settings:
+To specify the name and location of the target folder, pass a valid path (relative or absolute) to the `create-testcafe` command:
 
-**Type**: Boolean
-
-**Default value**: `true` unless the project directory already includes any project.
-
-**Alias**: `-w`
+```sh
+# npx
+npx create-testcafe appName
+# npm 
+# Preface the argument with a double dash
+npm init testcafe@latest -- appName
+# yarn
+yarn create testcafe appName
+# pnpm
+pnpm create testcafe appName
+```
 
-### template
+## Options
 
-The `--template` option determines the programming language for TestCafe tests.
+The `npm init` command handles options differently from other launch commands. See the [forward options to npm init](#forward-options-to-npm-init) section for more information.
 
-**Possible values**:
-* `javascript`
-* `typesctipt`
+| Option  | Description | Value | Default value 
+| ------------- | ------------- | ------------- | ------------- |
+| --run-wizard, -w | Launch the interactive wizard | Boolean  | `true` if the target folder is a Node.js project |
+| --template  | Project template  |`javascript` or `typescript` | `typescript` if the project directory includes a `.tsconfig.json` file. |
+| --test-folder  | Test subfolder path | Valid relative path | `test`|
+| --include-sample-tests  | Add sample tests to the project | Boolean | `true`|
+| --github-actions-init  | Add a GitHub Actions workflow file to the project | Boolean | `true`|
 
-**Default value**: `javascript` unless the project directory includes a `.tsconfig.json` file.
+## Forward options to npm init
 
-### test-folder
+The `npm init` command handles options and arguments differently from other launch commands. 
 
-The `--test-folder` option specifies the relative path to the test subfolder.
+If you run Windows, preface the folder location argument with a double dash. If you run macOS, you can omit the dash.
```

Comment:
The same: It is not necessary to add a double dash if only the path parameter is specified. I guess we can remove this part.